common/types/utils.go

```diff
@@ -63,3 +63,15 @@ func Intersection[T comparable](arrays ...[]T) []T {
 
 	return intersection
 }
+
+func RemoveDuplicatesFromSlice[T comparable](s []T) []T {
+	allKeys := make(map[T]bool)
+	list := []T{}
+	for _, item := range s {
+		if _, value := allKeys[item]; !value {
+			allKeys[item] = true
+			list = append(list, item)
+		}
+	}
+	return list
+}
```

Comment:
I believe this is unneeded now.

Reply:
done. see https://github.com/lavanet/lava/pull/577/commits/977d78fb6e68ecd92e94ac6b5192bc2f2f9aabc7